.nx/version-plans/version-plan-1749960117243.md

```diff
@@ -0,0 +1,5 @@
+---
+vincent-sdk: minor
```

Comment:
🔪 Since this is only an internal formatting change to docs, it wouldn't represent a semver change (no functionality has been added to the sdk package)

Reply:
@MaximusHaximus Removed the version plan